apps/shell/app.js

```diff
@@ -59,11 +60,8 @@ const server = new http.createServer(app);
 const wss = new WebSocket.Server({ noServer: true });
 
 let host_allowlist = new Set;
-if (process.env.OOD_SSHHOST_ALLOWLIST){
-  host_allowlist = new Set(process.env.OOD_SSHHOST_ALLOWLIST.split(':'));
-}
-
```

Comment:
As discussed offline, we want to revert the change here as that is unrelated to the support for wildcards.